lib/galaxy/jobs/runners/slurm.py

```diff
@@ -78,6 +91,8 @@ def _get_slurm_state():
                 if slurm_state == 'NOT_FOUND':
                     log.warning( '(%s/%s) Job not found, assuming job check exceeded MinJobAge and completing as successful', ajs.job_wrapper.get_id_tag(), ajs.job_id )
                     drmaa_state = self.drmaa_job_states.DONE
+                elif slurm_state == 'COMPLETED':
+                    drmaa_state = self.drmaa_job_states.DONE
```

Comment:
Can we log something here? I think the case that DRMAA reports failure but Slurm reports success and thus Galaxy reports success should probably be noted somehow.